<?xml version="1.0" encoding="UTF-8"?>
<TestRun id="ed9839e4-6e70-4502-b63a-04e3d4d03800" name="Fredrik@FKVAIO 2010-09-13 14:09:09" runUser="FKVaio\Fredrik" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <TestSettings name="Local" id="5b82bb5d-938d-47a6-8c5b-afaf40dc0149">
    <Description>These are default test settings for a local test run.</Description>
    <Deployment enabled="false" runDeploymentRoot="Fredrik_FKVAIO 2010-09-13 14_09_09" />
    <Execution>
      <TestTypeSpecific />
      <AgentRule name="Execution Agents">
      </AgentRule>
    </Execution>
  </TestSettings>
  <Times creation="2010-09-13T14:09:09.8391745+02:00" queuing="2010-09-13T14:09:10.1321913+02:00" start="2010-09-13T14:09:10.3572042+02:00" finish="2010-09-13T14:09:11.4012639+02:00" />
  <ResultSummary outcome="NotRunnable">
    <Counters total="6" executed="5" passed="5" error="0" failed="0" timeout="0" aborted="0" inconclusive="0" passedButRunAborted="0" notRunnable="1" notExecuted="0" disconnected="0" warning="0" completed="0" inProgress="0" pending="0" />
  </ResultSummary>
  <TestDefinitions>
    <UnitTest name="TaskPrioritised_event_is_published" storage="c:\source\_projects\nexttask\nexttask.tests\bin\debug\nexttask.tests.dll" id="e15260cc-8d7d-d0e2-46e8-7391435d2c0e">
      <Execution timeOut="1800000" id="59491b0e-6afb-4e98-8bf4-d8cae44e1bf0" />
      <TestMethod codeBase="C:/Source/_Projects/NextTask/NextTask.Tests/bin/Debug/NextTask.Tests.DLL" adapterTypeName="Microsoft.VisualStudio.TestTools.TestTypes.Unit.UnitTestAdapter, Microsoft.VisualStudio.QualityTools.Tips.UnitTest.Adapter, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" className="NextTask.Tests.When_prioritising_task, NextTask.Tests, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null" name="TaskPrioritised_event_is_published" />
    </UnitTest>
    <UnitTest name="TaskPostponed_event_is_published" storage="c:\source\_projects\nexttask\nexttask.tests\bin\debug\nexttask.tests.dll" id="48190fb3-47e2-4c5f-b51b-b8f8d829603b">
      <Execution timeOut="1800000" id="e065ea2f-7aee-4508-a311-949e7b753218" />
      <TestMethod codeBase="C:/Source/_Projects/NextTask/NextTask.Tests/bin/Debug/NextTask.Tests.DLL" adapterTypeName="Microsoft.VisualStudio.TestTools.TestTypes.Unit.UnitTestAdapter, Microsoft.VisualStudio.QualityTools.Tips.UnitTest.Adapter, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" className="NextTask.Tests.When_postponing_task, NextTask.Tests, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null" name="TaskPostponed_event_is_published" />
    </UnitTest>
    <UnitTest name="Exception_is_thrown" storage="c:\source\_projects\nexttask\nexttask.tests\bin\debug\nexttask.tests.dll" id="98bb3bc4-05d4-f8e2-a91b-ad8e68c6fd76">
      <Execution timeOut="1800000" id="8756758e-58ee-4d6d-a126-7b2f09408a91" />
      <TestMethod codeBase="C:/Source/_Projects/NextTask/NextTask.Tests/bin/Debug/NextTask.Tests.DLL" adapterTypeName="Microsoft.VisualStudio.TestTools.TestTypes.Unit.UnitTestAdapter, Microsoft.VisualStudio.QualityTools.Tips.UnitTest.Adapter, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" className="NextTask.Tests.When_postponing_task_with_new_due_date_less_than_existing_due_date, NextTask.Tests, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null" name="Exception_is_thrown" />
    </UnitTest>
    <UnitTest name="TaskCancelled_event_is_published" storage="c:\source\_projects\nexttask\nexttask.tests\bin\debug\nexttask.tests.dll" id="7a533ce9-67f5-f9af-ad97-121a2f49f974">
      <Execution timeOut="1800000" id="f46747e2-b502-4c67-8fd6-f39ee99fd936" />
      <TestMethod codeBase="C:/Source/_Projects/NextTask/NextTask.Tests/bin/Debug/NextTask.Tests.DLL" adapterTypeName="Microsoft.VisualStudio.TestTools.TestTypes.Unit.UnitTestAdapter, Microsoft.VisualStudio.QualityTools.Tips.UnitTest.Adapter, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" className="NextTask.Tests.When_completing_task, NextTask.Tests, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null" name="TaskCancelled_event_is_published" />
    </UnitTest>
    <UnitTest name="TaskCancelled_event_is_published" storage="c:\source\_projects\nexttask\nexttask.tests\bin\debug\nexttask.tests.dll" id="28c8e7d6-deb0-1a4d-e172-64b990106212">
      <Execution timeOut="1800000" id="ed096f66-50bc-42c9-b10c-f040854f61f0" />
      <TestMethod codeBase="C:/Source/_Projects/NextTask/NextTask.Tests/bin/Debug/NextTask.Tests.DLL" adapterTypeName="Microsoft.VisualStudio.TestTools.TestTypes.Unit.UnitTestAdapter, Microsoft.VisualStudio.QualityTools.Tips.UnitTest.Adapter, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" className="NextTask.Tests.When_cancelling_task, NextTask.Tests, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null" name="TaskCancelled_event_is_published" />
    </UnitTest>
    <UnitTest name="Exception_is_thrown" storage="c:\source\_projects\nexttask\nexttask.tests\bin\debug\nexttask.tests.dll" id="2b08e450-89fa-4f49-348f-b842a68ad041">
      <Execution id="c77260d1-a2b4-4f21-a940-ef9ebed55679" />
      <TestMethod codeBase="C:/Source/_Projects/NextTask/NextTask.Tests/bin/Debug/NextTask.Tests.DLL" adapterTypeName="Microsoft.VisualStudio.TestTools.TestTypes.Unit.UnitTestAdapter, Microsoft.VisualStudio.QualityTools.Tips.UnitTest.Adapter, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" className="NextTask.Tests.When_cancelling_task_that_has_already_been_completed, NextTask.Tests, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null" name="Exception_is_thrown" />
    </UnitTest>
  </TestDefinitions>
  <TestLists>
    <TestList name="Results Not in a List" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="All Loaded Results" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <TestEntries>
    <TestEntry testId="28c8e7d6-deb0-1a4d-e172-64b990106212" executionId="ed096f66-50bc-42c9-b10c-f040854f61f0" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestEntry testId="2b08e450-89fa-4f49-348f-b842a68ad041" executionId="c77260d1-a2b4-4f21-a940-ef9ebed55679" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestEntry testId="7a533ce9-67f5-f9af-ad97-121a2f49f974" executionId="f46747e2-b502-4c67-8fd6-f39ee99fd936" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestEntry testId="48190fb3-47e2-4c5f-b51b-b8f8d829603b" executionId="e065ea2f-7aee-4508-a311-949e7b753218" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestEntry testId="98bb3bc4-05d4-f8e2-a91b-ad8e68c6fd76" executionId="8756758e-58ee-4d6d-a126-7b2f09408a91" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestEntry testId="e15260cc-8d7d-d0e2-46e8-7391435d2c0e" executionId="59491b0e-6afb-4e98-8bf4-d8cae44e1bf0" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <Results>
    <UnitTestResult executionId="ed096f66-50bc-42c9-b10c-f040854f61f0" testId="28c8e7d6-deb0-1a4d-e172-64b990106212" testName="TaskCancelled_event_is_published" computerName="FKVAIO" duration="00:00:00.7457386" startTime="2010-09-13T14:09:10.4372087+02:00" endTime="2010-09-13T14:09:11.2812570+02:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Passed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="ed096f66-50bc-42c9-b10c-f040854f61f0">
    </UnitTestResult>
    <UnitTestResult executionId="c77260d1-a2b4-4f21-a940-ef9ebed55679" testId="2b08e450-89fa-4f49-348f-b842a68ad041" testName="Exception_is_thrown" computerName="FKVAIO" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="NotRunnable" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="c77260d1-a2b4-4f21-a940-ef9ebed55679">
      <Output>
        <ErrorInfo>
          <Message>UTA004: Illegal use of attribute on NextTask.Tests.When_cancelling_task_that_has_already_been_completed.Exception_is_thrown. The TestMethodAttribute can be defined only inside a class marked with the TestClass attribute.</Message>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
    <UnitTestResult executionId="f46747e2-b502-4c67-8fd6-f39ee99fd936" testId="7a533ce9-67f5-f9af-ad97-121a2f49f974" testName="TaskCancelled_event_is_published" computerName="FKVAIO" duration="00:00:00.0062456" startTime="2010-09-13T14:09:11.2902575+02:00" endTime="2010-09-13T14:09:11.2972579+02:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Passed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="f46747e2-b502-4c67-8fd6-f39ee99fd936">
    </UnitTestResult>
    <UnitTestResult executionId="e065ea2f-7aee-4508-a311-949e7b753218" testId="48190fb3-47e2-4c5f-b51b-b8f8d829603b" testName="TaskPostponed_event_is_published" computerName="FKVAIO" duration="00:00:00.0081041" startTime="2010-09-13T14:09:11.3012582+02:00" endTime="2010-09-13T14:09:11.3112587+02:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Passed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="e065ea2f-7aee-4508-a311-949e7b753218">
    </UnitTestResult>
    <UnitTestResult executionId="8756758e-58ee-4d6d-a126-7b2f09408a91" testId="98bb3bc4-05d4-f8e2-a91b-ad8e68c6fd76" testName="Exception_is_thrown" computerName="FKVAIO" duration="00:00:00.0035471" startTime="2010-09-13T14:09:11.3132589+02:00" endTime="2010-09-13T14:09:11.3222594+02:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Passed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="8756758e-58ee-4d6d-a126-7b2f09408a91">
    </UnitTestResult>
    <UnitTestResult executionId="59491b0e-6afb-4e98-8bf4-d8cae44e1bf0" testId="e15260cc-8d7d-d0e2-46e8-7391435d2c0e" testName="TaskPrioritised_event_is_published" computerName="FKVAIO" duration="00:00:00.0056639" startTime="2010-09-13T14:09:11.3242595+02:00" endTime="2010-09-13T14:09:11.3312599+02:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Passed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="59491b0e-6afb-4e98-8bf4-d8cae44e1bf0">
    </UnitTestResult>
  </Results>
</TestRun>